src/storage/mod.rs

```diff
@@ -130,7 +130,11 @@ pub enum Command {
         keys: Vec<Key>,
         start_ts: u64,
     },
-    ScanLock { ctx: Context, max_ts: u64 },
+    ScanLock {
+        ctx: Context,
+        max_ts: u64,
+        limit: Option<usize>,
```

Comment:
why Option here? If limit is 0, you can skip it.

Reply:
no need to use it, I will remove all Option about `limit`